src/System.Linq/src/System.Linq.csproj

```diff
@@ -3,6 +3,7 @@
     <ProjectGuid>{CA488507-3B6E-4494-B7BE-7B4EEEB2C4D1}</ProjectGuid>
     <AssemblyName>System.Linq</AssemblyName>
     <RootNamespace>System.Linq</RootNamespace>
+    <NoWarn>$(NoWarn);CS1574</NoWarn>
```

Comment:
❕ I would prefer to see either of the following approaches:

1. Suppress CS1574 at the specific site(s) where it occurs, and not for the entire project
2. Use the ``T:Some.Namespace.ConcatIterator`1`` syntax instead of relying on the compiler to resolve the reference

Reply:
In this case I prefer the second option. Thank you for the suggestion.